index.html

```diff
@@ -61,11 +66,12 @@ <h1 class="logo">My Logo</h1>
        <section class="parting">
         <div class="tonic">
```

Comment:
Don't you think some space between the cards will make it more beautiful? So please put some space for it.

![image](https://user-images.githubusercontent.com/34779928/194648474-4ffe1037-71c0-47f1-a69f-f9ec0233cbda.png)